src/components/Board.js

```diff
@@ -13,11 +13,12 @@ const Board = (props) => {
   const [cards, setCards] = useState([]);
   const [errorMessage, setErrorMessage] = useState(null);
 
+  //populate cards
   useEffect(() => {
-    axios
-      .get(API_URl_BASE)
+    axios.get(API_URl_BASE + "/cards") 
       .then((res) => {
         const apiCardsData = res.data;
+        console.log(res)
```

Comment:
I did a console.log here just to see how the data was returning, will take out when we clean up the code.